pkg/controllers/cspc-controller/scale.go

```diff
@@ -68,10 +68,8 @@ func (pc *PoolConfig) CreateCSPI(cspc *cstor.CStorPoolCluster) error {
 		return err
 	}
 	err = pc.CreateCSPIDeployment(cspc, cspi)
```

Comment:
nit: Even we can do in the following way
```suggestion
      return pc.CreateCSPIDeployment(cspc, cspi)
```